wmn-data.json

```diff
@@ -974,10 +974,10 @@
        },
        {
         "name" : "Chess.com",
-        "uri_check" : "https://www.chess.com/member/{account}",
+        "uri_check" : "https://api.chess.com/pub/player/{account}",
         "e_code" : 200,

```

Comment:
Thanks for switching this. When we use an API for the check, we always try to add a `uri+pretty` that an app can point a user to. Can you just add the previous `uri_check` as a `uri_pretty`?